web/src/features/PreflightChecks/api/postPreflightRun.tsx

```diff
@@ -4,22 +4,24 @@ async function postPreflightRun({
   apiEndpoint = process.env.API_ENDPOINT,
   slug,
   sequence,
+  isUpgradeService = false,
```

Comment:
i'd like for the upgrade service components to be isolated from non-upgrade service components in order to make it easier to rip out this code in the future when everything goes through the upgrade service. it's fine to duplicate code to achieve that. i'd like to avoid conditionals like this.

Reply:
that makes sense. I've extracted it to it's own hook inside of upgrade-service